package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "npm run clean && babel src --out-dir lib --ignore *-test.js",
     "clean": "rimraf lib",
-    "lint": "esw src --color",
+    "lint": "esw src --color --fix",
```

Comment:
Will you remove this later or can we separate this into another command called `lint:fix`? We need this command without auto-fix for CI.